aci/resource_aci_bgpprotp.go

```diff
@@ -226,32 +276,47 @@ func resourceAciL3outBGPProtocolProfileRead(ctx context.Context, d *schema.Resou
 		d.SetId("")
 		return nil
 	}
+
+	bgpRsBestPathCtrlPolData, err := aciClient.ReadRelationbgpRsBestPathCtrlPol(dn)
+	if err != nil {
+		log.Printf("[DEBUG] Error while reading relation bgpRsBestPathCtrlPol %v", err)
+		d.Set("relation_bgp_rs_best_path_ctrl_pol", "")
+	} else {
+		if _, ok := d.GetOk("relation_bgp_rs_best_path_ctrl_pol"); ok {
+			tfName := GetMOName(d.Get("relation_bgp_rs_best_path_ctrl_pol").(string))
+			if tfName != bgpRsBestPathCtrlPolData {
```

Comment:
I am confused on what you are trying to do here. tfName is not clear enough as a variable name for this relationship dn attribute.